Makefile

```diff
@@ -139,7 +139,7 @@ endif
 install: build
 	# binaries
 	mkdir -p "$${PREFIX}/bin"
-	cp "$${VTROOTBIN}/"{mysqlctl,mysqlctld,vtorc,vtadmin,vtctld,vtctlclient,vtctldclient,vtgate,vttablet,vtbackup} "$${PREFIX}/bin/"
+	cp "$${VTROOTBIN}/"{mysqlctl,mysqlctld,vtorc,vtadmin,vtctl,vtctld,vtctlclient,vtctldclient,vtgate,vttablet,vtbackup,vtexplain} "$${PREFIX}/bin/"
```

Comment:
The image `vitess/k8s` now relies on `vitess/lite`, which runs `make install`. In `vitess/k8s` we copy these binaries so they are available for the binary-specific images. I had to add the missing binaries that we use in `vitess/k8s`.